doc/source/examples/cd_mmd_keops.ipynb

```diff
@@ -0,0 +1,513 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plot_relative_time(results, [2], max_batch_size=max_batch_size)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f7dc206c",
+   "metadata": {},
+   "source": [
+    "## Conclusion\n",
+    "\n",
+    "As illustrated in the experiments, KeOps allows you to drastically speed up and scale up drift detection to larger datasets without running into memory issues. The speed benefit of KeOps over the PyTorch (or TensorFlow) MMD detector decreases as the number of features increases. Note though that it is not advised to apply the (untrained) MMD detector to very high-dimensional data in the first place."
+   ]
+  }
+ ],
+ "metadata": {
+  "kernelspec": {
+   "display_name": "Python [conda env:detect]",
+   "language": "python",
+   "name": "conda-env-detect-py"
```

Comment:
This causes an error in the notebook CI.

Reply:
`display_name` should also be reset to the default (either `Python 3` or `Python 3 (ipykernel)`) to not cause an error for a user opening the notebook.